frontend/mono_frontend/.gitignore

```diff
@@ -5,6 +5,7 @@
 package-lock.json
 /.pnp
 .pnp.js
+../.prettierrc
```

Comment:
I would keep this file in repo to have the same prettier config in the team

Reply:
Done